README.md

```diff
@@ -9,9 +9,9 @@
 <p align="center">
   <a href="https://github.com/ethereum-optimism/kona/actions/workflows/rust_ci.yaml"><img src="https://github.com/ethereum-optimism/kona/actions/workflows/rust_ci.yaml/badge.svg?label=ci" alt="CI"></a>
   <a href="https://github.com/ethereum-optimism/kona/actions/workflows/fpvm_tests.yaml"><img src="https://github.com/ethereum-optimism/kona/actions/workflows/fpvm_tests.yaml/badge.svg?label=FPVM Tests" alt="FPVM Tests"></a>
-  <img src="https://img.shields.io/badge/License-MIT-green.svg?label=license" alt="License">
-  <a href="https://ethereum-optimism.github.io/kona"><img src="https://img.shields.io/badge/Contributor%20Book-grey?logo=mdBook" alt="Book"></a>
-  <a href="https://github.com/ethereum-optimism/monorepo"><img src="https://img.shields.io/badge/OP%20Stack-monorepo-red" alt="OP Stack"></a>
+    <a href="https://crates.io/crates/kona-derive"><img src="https://img.shields.io/crates/v/kona-derive.svg?label=kona-derive&labelColor=2a2f35" alt="Kona Derive"></a>
```

Comment:
Can we put the crate links in the individual crates? `kona-derive` is a smaller part of the broader workspace.

Reply:
Definitely, good thinking.